### PR TITLE
[RU] adding RU support + introduction page

### PR DESCRIPTION
**Pull request by ohgodnotagain**

### CODE REVIEW

**Comment by DarthGandalf** [comment]
Why did you need to close the previous PR?

**Review by DarthGandalf — APPROVED** [review]

**Review by qwrtln — CHANGES_REQUESTED** [review]
Please make the change suggested by @DarthGandalf in the previous PR regarding `\providecommand`. Also, add `ru` to README in line 76.

**Comment by DarthGandalf** [comment]
What about 6bc51eb ?

**Comment by DarthGandalf** [comment]
I'm playing with some fonts, does this or this look any better?

изображение
изображение

I'd like first font for text, but second font for headers, but I can't find how to combine them in a single document

**Comment by DarthGandalf** [comment]
That's possible by switching to TTF fonts using

```latex
\usepackage[TU]{fontenc}
\usepackage{fontspec}
\setmainfont{Arial}
\newfontfamily\titlefont{Comic Sans MS}
```
Then using \titlefont (or whatever name was chosen) in \addsection

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Click in the table to download the most recent builds in the chosen language:
...
 | 🇪🇸 Español    |~25%|[🇪🇸📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/es/main_es.pdf)|[🇪🇸🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/es/printable_es.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_es/components_list_es.pdf)|
 | 🇫🇷 Français   |~10%|[🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/main_fr.pdf)|[🇫🇷🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/printable_fr.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_fr/components_list_fr.pdf)|
+| 🇷🇺 Русский   |~1%|[🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/main_ru.pdf)|[🇷🇺🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/printable_ru.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_ru/components_list_ru.pdf)|
```
Comment (qwrtln) [inline README.md:16]:
```suggestion
| 🇷🇺 Русский   |~1%|[🇷🇺📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/main_ru.pdf)|[🇷🇺🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/printable_ru.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_ru/components_list_ru.pdf)|
```

**Review by qwrtln — CHANGES_REQUESTED** [review]
Please generate all the `ru.po` files, so that someone doesn't get confused on their PR.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Click in the table to download the most recent builds in the chosen language:
...
 | 🇫🇷 Français   |~10%|[🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/main_fr.pdf)|[🇫🇷🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/printable_fr.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_fr/components_list_fr.pdf)|
-| 🇷🇺 Русский   |~1%|[🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/main_ru.pdf)|[🇷🇺🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/printable_ru.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_ru/components_list_ru.pdf)|
+| 🇷🇺 Русский   |~5%|[🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/main_ru.pdf)|[🇷🇺🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/printable_ru.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_ru/components_list_ru.pdf)|
```
Comment (qwrtln) [inline README.md:16]:
```suggestion
| 🇷🇺 Русский   |~5%|[🇷🇺📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/main_ru.pdf)|[🇷🇺🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/ru/printable_ru.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_ru/components_list_ru.pdf)|
```
The flag! Suggestion had it too.

**Review by qwrtln — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**